angular/projects/admin-nrpti/src/app/mines/mines-collection-detail/mines-collection-detail.component.html

```diff
@@ -0,0 +1,74 @@
+<main class="container-fluid-padding">
+  <section>
+    <div class="d-flex flex-column flex-sm-row justify-content-sm-between heading-bar">
+      <div>
+        <h1 id="name" class="m-0">{{ (collection && collection.name) || '-' }}</h1>
+      </div>
+      <div>
+        <span ngbDropdown>
+          <button class="btn btn-primary action-drop-button" id="actionDropdown" ngbDropdownToggle>Actions</button>
+          <div class="dropdown-menu" aria-labelledby="actionDropdown" ngbDropdownMenu>
+            <button class="dropdown-item" (click)="navigateToEditPage()">Edit</button>
+          </div>
+        </span>
+      </div>
+    </div>
+  </section>
+
+  <section class="mb-3 pb-3 border-bottom">
+    <div class="row">
+      <div class="mb-2 col-12 col-sm-8">
+        <h2 class="border-0 mb-0">Collection Information</h2>
+        <span class="grey-subtext">{{ lastEditedSubText }}</span>
+      </div>
+      <div class="col-12 col-sm-4">
+        <div class="d-flex flex-column">
+          <mat-slide-toggle
+            name="publishLng"
+            id="publishLng"
+            class="publish-toggle"
+            [ngClass]="isPublished ? 'published' : 'unpublished'"
+            [checked]="isPublished"
+            [disabled]="true"
```

Comment:
Did they want the ability to publish collections from this view?
Or possibly this is just waiting on other collections tickets to get finished?

Reply:
It is disabled as it was just to show if this is published or not.  This action is actually done in the edit screen.